release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2077,4 +2077,31 @@ A new conditional gatherer has been implemented for the `SamplesImagestreamImpor
 [id="ocp-4-9-4-upgrading"]
 ==== Upgrading
 
-To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-upgrading-cli_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
```

Comment:
I believe this should remain 4.9. All prior release notes have stated the current 4.y version there. I believe we're targeting earlier z-stream release upgrades to the current 4.y version z-stream.